src/groups/group.jl

```diff
@@ -21,6 +21,29 @@ Note that a manifold is connected with an operation by wrapping it with a decora
 """
 abstract type AbstractGroupOperation end
 
+"""
+    abstract type AbstractGroupDecroatorType <: AbstractDecoratorType
+
+    A common decorator type for all group decorators
+"""
+abstract type AbstractGroupDecoratorType <: AbstractDecoratorType end
+
+"""
+    struct DefaultGroupDecoratorType <: AbstractDecoratorType
+
+A default group decorator type with no special properties
```

Comment:
```suggestion
The default group decorator type with no special properties.
```
How exactly does this differ from `TransparentGroupDecoratorType`?